universal-application-tool-0.0.1/app/views/applicant/ApplicantInformationView.java

```diff
@@ -62,6 +64,7 @@ public Content render(
             .setTitle("Applicant information")
             .addMainStyles(ApplicantStyles.MAIN_APPLICANT_INFO)
             .addMainContent(formContent);
+    bundle.addMainContent(h1("Applicant information").withClasses(Styles.SR_ONLY));
```

Comment:
suggestion (blocking): instead of a raw string, use the i18n system and add the english text to the messages file